airbyte-integrations/connectors/source-chartmogul/source_chartmogul/source.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from base64 import b64encode
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+
+
+# Basic full refresh stream
+class ChartmogulStream(HttpStream, ABC):
+    url_base = "https://api.chartmogul.com"
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        yield from response.json().get("entries", [])
+
+
+class Customers(ChartmogulStream):
+    primary_key = "uuid"
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        json_response = response.json()
+        if not json_response.get("has_more", False):
+            return None
+
+        return {"page": json_response.get("current_page") + 1}
```

Comment:
nit, for readability:
```suggestion
        if json_response.get("has_more", False):
            return {"page": json_response.get("current_page") + 1}

        return None
```